### PR TITLE
Fix linter errors

### PR DESCRIPTION
**Pull request by jacquelinekay**

### CODE REVIEW

**Comment by jacquelinekay** [comment]
Here's a Connext Dynamic-only set of CI jobs:

[Build Status](http://ci.ros2.org/job/ci_linux/1633/)
[Build Status](http://ci.ros2.org/job/ci_osx/1227/)

File: rmw_connext_dynamic_cpp/src/macros.hpp [hunk 1]
```diff
@@ -84,7 +86,7 @@
     } \
   }; \
   template<> \
-  struct IdTypeMap<rosidl_typesupport_introspection_c__ROS_TYPE_ ## ROS_TYPE_ID> { \
+  struct IdTypeMap<rosidl_typesupport_introspection_c__ROS_TYPE_ ## ROS_TYPE_ID>{ \
```
Comment (wjwwood) [inline rmw_connext_dynamic_cpp/src/macros.hpp]: hrm, that's annoying...

**Comment by wjwwood** [comment]
code changes lgtm

**Comment by wjwwood** [comment]
I'm going to merge this to get rid of the lint failures on master.

**Comment by jacquelinekay** [comment]
I was looking into some connext dynamic failures in those CI jobs. But I'll try to fix them in a subsequent pr.

**Comment by wjwwood** [comment]
Oh, sorry, I didn't realize. I think this was a strict improvement though right?

**Comment by jacquelinekay** [comment]
No, based on some local testing with different states before and after this change, I think I introduced a subtle bug here: https://github.com/ros2/rmw_connext/pull/191/files#diff-a1342c2267183e68decd53f4e4273732R1239

I'll have another patch in a sec.

**Pull request merged**